### PR TITLE
Fix unthemed DIVs in JIRA Epic issues list (atlassian.net)

### PR DESCRIPTION
**Pull request by terminalmage**
NOTE: The other changes in this file are due to a vim autocmd I have to remove trailing whitespace on buffer write.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1557,6 +1557,9 @@ code:first-of-type {
     box-shadow: 0 0 4px ${rgba(3,9,15,0.15)} !important;
     background-color: ${rgba(3, 9, 15, 0.02)} !important;
 }
+.sc-186t4q4-1 {
+    background-color: rgba(3, 9, 15, 0.02) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1561]: Could this like the others ones in the form of `${original color}`?

Comment (terminalmage) [reply src/config/dynamic-theme-fixes.config:1561]: I don't understand the question. Are you asking if it should be wrapped in `${}`?

To be honest, I'm not even sure what that does, CSS is not one of my strengths.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1561]: > Are you asking if it should be wrapped in `${}`?
> 
> To be honest, I'm not even sure what that does, CSS is not one of my strengths.

Yes I'm asking that. I'm sorry to not be upfront by this, but this is something Dark Reader-specific. It allows to invert the color on-the-fly given the user's setting. Thereby we don't rely on hard coded values(which won't work on light mode + custom settings).

Comment (terminalmage) [reply src/config/dynamic-theme-fixes.config]: That wasn't the original color, that was one I picked, using the value from the similarly-named class above it.

Here is the CSS for this DIV:

image

The DIV has a class of `sc-186t4q4-1 kTpmQd`.

**Review by Gusted — APPROVED** [review]

**Pull request merged**